src/azul/service/index_query_service.py

```diff
@@ -156,12 +157,12 @@ def get_summary(self, catalog: CatalogName, filters):
                 'speciesCount'
             ],
             'projects': [
-                'projectCount',
-                'labCount',
+                'project',
+                'labCount'
             ],
             'cell_suspensions': [
-                'totalCellCount',
-                'cellCountSummaries',
+                'total_cell_count',
+                'group_by_organ'
```

Comment:
See above. There is no good reason that the aggregation is named using snake while the response element is named camel.